examples/statics/kibana-links.yaml

```
@@ -0,0 +1,51 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kibana-links
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: kibana-links
+    busola.io/extension: statics
+    busola.io/extension-version: '0.5'
+data:
+  injections: |-
+    - name: "Logs for namespace"
+      source: "'Discover Logs'"
+      link: "$prepareUrl(
+        'https://kibana-sf-a7ac93be-d8bd-43d1-b85a-dcf3bf46c470.cls-01.cloud.logs.services.sap.hana.ondemand.com/app/discover?security_tenant=global#/?_g=(filters:!(),refreshInterval:(pause:!f,value:5000),time:(from:now-7d,to:now))&_a=(columns:!(kubernetes.pod_name,kubernetes.container_name,log),filters:!((query:(match_phrase:(kubernetes.namespace_name:{namespace_name})))),index:maintained-by-perfx_kyma-content-package_index-pattern-logs-json-kyma)', 
+        {'namespace_name': $embedResource.metadata.name}
+      )"
+      widget: ExternalLink
+      type: button
+      copyable: false
+      targets:
+        - slot: details-header
+          location: Namespaces
+      order: 1 
+    - name: "Logs for pod"
```

```suggestion
    - name: "Logs for Pod"
```
should we capitalize it here as in our docs?